src/ovirtapi.js

```diff
@@ -317,6 +330,7 @@ OvirtApi = {
       os: {
         type: template.os ? template.os.type : undefined,
       },
+      ...cloudInitApiToInternal(template),
```

Comment:
For the sake of consistency, can you change to:
```
cloudInit: cloudInitApiToInternal(template)
```
and in `cloudInitApiToInternal()` return just the props of the object?

If agreed, this can be done in a follow-up (but shortly once this is merged :-) )

Reply:
done